shell-completion/bash/firewall-offline-cmd

```diff
@@ -0,0 +1,228 @@
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_IPSET_ACTION_ACTION="--add-entry= --remove-entry= --query-entry= \
+                             --add-entries-from-file= --remove-entries-from-file"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+## OBS: icmp-block-inversion only in the new release 0.6.0.0
+OPTIONS_IPSET_ADAPT_QUERY="--list-entries"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+# can be used with/without preceding --zone=<zone>
+OPTIONS_ZONE="${OPTIONS_ZONE_INTERFACES_SOURCES} \
+              ${OPTIONS_ZONE_ACTION_ACTION} ${OPTIONS_ZONE_ADAPT_QUERY} \
+              ${OPTIONS_ZONE_PERMANENT_ONLY}"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_IPSET="${OPTIONS_IPSETACTION_ACTION} ${OPTIONS_IPSET_ADAPT_QUERY}"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_PERMANENT_ONLY="--new-icmptype= --new-icmptype-from-file= --delete-icmptype= \
+                        --new-service= --new-service-from-file= --delete-service= \
+                        --new-zone= --new-zone-from-file= --delete-zone= \
+                        --new-ipset= --new-ipset-from-file= --delete-ipset= \
+                        --new-helper= --new-helper-from-file= --delete-helper= \
+                        --get-target --set-target= \
+                        --path-zone= --path-service= --path-icmptype= \
+                        --path-ipset= --path-helper="
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_NEW_IPSET="--type= --option="
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_NEW_HELPER="--module= --family="
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_HELPER=""
+
+## Revisado todas as opções --DFR-- 30-07-2018
+# can be used after --permanent
+OPTIONS_PERMANENT="${OPTIONS_CONFIG} --zone= ${OPTIONS_ZONE} \
+                   ${OPTIONS_PERMANENT_ONLY}"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+OPTIONS_DIRECT=" --add-chain --remove-chain --query-chain --get-chains --get-all-chains \
+         --add-rule --remove-rule  --remove-rules --query-rule  --get-rules  --get-all-rules \
+         --add-passthrough --remove-passthrough \
+         --query-passthrough --get-passthroughs --get-all-passthroughs"
+
+## Revisado todas as opções --DFR-- 30-07-2018
+# these all can be used as a "first" option
+OPTIONS_GENERAL="--help --version \
+                 --get-log-denied --set-log-denied= \
+                 --lockdown-on --lockdown-off --query-lockdown \
+                 --get-default-zone --set-default-zone= \
+                 --get-zone-of-interface= \
+                 ${OPTIONS_CONFIG} \
+                 --zone= ${OPTIONS_ZONE} \
+                 --direct"
+
+_firewall_offline_cmd()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+    firewall-cmd --state &> /dev/null
+    if [[ "${?}" -eq 0 ]]; then
+        echo -e "\nfirewalld is running, please use firewall-cmd."
+        return
```

Comment:
this doesn't seem right. `firewall-offline-cmd` may be for daemon-less use, but it should not fail because the daemon is running.